lib/vets/attributes/value.rb

```diff
@@ -0,0 +1,53 @@
+module Vets
+  module Attributes
+    class Value
+
+      def self.cast(name, klass, value, array: false)
+        new((name, klass, value, array)).setter_value(value)
+      end
+
+      def initialize(name, klass, array: false)
+        @name = name
+        @klass_type = klass_type
+        @array = array
```

Comment:
Could'nt this be inferred from its type?

Reply:
That's a mistake it should be just `@klass`